### PR TITLE
Bump selenium version to 4.1.4

### PR DESCRIPTION
**Pull request by kdaud**
Upgrade selenium version to 4.1.4 to attempt fixing the current Bamboo failure.

### CODE REVIEW

**Comment by sherrif10** [comment]
Does it break the ci, is it helping us to resolve bamboo

**Comment by kdaud** [comment]
The failures at Bamboo are too generic that can not give the right cause why all the tests are failing. I think upgrading the selenium version to the latest will give a better Bamboo logs that could be helpful.

Besides, library upgrade are so good to have in our module and am wondering if the we could have openmrs-bot configured to have it help us in upgrading the module dependencies.

File: qaframework-bdd-tests/pom.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
 		<dependency>
 			<groupId>org.seleniumhq.selenium</groupId>
 			<artifactId>selenium-java</artifactId>
-			<version>4.3.0</version>
+			<version>4.1.3</version>
```
Comment (sherrif10) [inline qaframework-bdd-tests/pom.xml:50]: Any Reason of reverting it back ?

Comment (kdaud) [reply qaframework-bdd-tests/pom.xml:50]: Am just testing out something but will get it back to the latest version.

Comment (kdaud) [reply qaframework-bdd-tests/pom.xml]: Any version higher than 4.1.4 breaks all the tests so have instead upgraded to 4.1.4

**Pull request closed without merging**